### ISSUE TITLE
Enhance reference of blend() and Image.blend() and make it consistent with blendMode()

### ISSUE THREAD

**Issue by membralala** [processing/p5.js#7236]
### Increasing access

Comprehensive examples of blend modes will provide a quick overview for people who are not used to work with image blending/layering.

### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [X] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### Feature enhancement details

I noticed a few small things, that might improve the documentation consistency between [blend()](https://p5js.org/reference/p5/blend/), [Image.blend()](https://p5js.org/reference/p5.Image/blend/) and [blendMode()](https://p5js.org/reference/p5/blendMode/): 

- `blendMode()` provides a comprehensive overview of all blend modes, while `blend()` and `Image.blend()` don't. However it might help people without experience on image blending to see, how other blend modes affect the images as well. 
- I noticed, that `blend()` and `Image.blend()` do not mention `REMOVE` as blend mode. However, as far as I tested it, it seems to work and acts as an inverse mask, if an image with alpha channel is provided (see example below). I wonder, if this behaviour is intentional, but if so, it probably should be documented as well. 
 ```
let photo;
let maskImage;

// Load the images.
function preload() {
  photo = loadImage('/assets/rockies.jpg');
  maskImage = loadImage('/assets/mask2.png');
}

function setup() {
  createCanvas(100, 100);

  // Apply the mask.
  photo.blend(maskImage, 0, 0, 100, 100, 0, 0, 100, 100, REMOVE);

  // Display the image.
  image(photo, 0, 0);

  describe('A white canvas. The right side of the canvas has a faded patch of an image of a mountain landscape.');
``` 
<img width="99" alt="Bildschirmfoto 2024-09-05 um 17 49 02" src="https://github.com/user-attachments/assets/1624ad6f-962d-4d0d-91f9-2b081d707c44">

- Also, the parameter reference of `Image.blend()` contains remnants of an outdated documentation including a broken link.

**Comment by swastikCommits** [comment]
Can I work on this issue?

**Comment by ksen0** [comment]
Hi @swastikCommits are you still interested in working on this?
Adding here documentation steward @perminder-17 - is this still applicable? on 1.x or 2.x or both? Thank you!

**Comment by perminder-17** [comment]
You are correct @membralala , `blend()` and `image.blend()` docs needs to be updated exactly what you said on the issue. Really thanks for catching this and explaining it so well. 

I think @ksen0 it's applicable in both our versions, 1.x and 2.x. 

If @swastikCommits you are interested I can assign you with the issue?

**Comment by swastikCommits** [comment]
Yeah, sure I would like to work on this issue!

**Comment by perminder-17** [comment]
> Yeah, sure I would like to work on this issue!

Sure, I have assigned you this. Thank you so much for taking this 🙂

**Comment by vaishnavirrapolu-06** [comment]
Hi! I'm Vaishnavi Rapolu, a student developer from Mumbai. I've been following the p5.js 2.0 transition and would love to help by making the documentation for blend() more consistent with blendMode(). As I'm new to open source, I'm excited to start with this documentation task. May I be assigned to this?

**Comment by perminder-17** [comment]
Since the issue was assigned a long time ago, I'll reassign it to you.

**Comment by vaishnavirrapolu-06** [comment]
Hi! @perminder-17 I have done the changes can you please go through it..
